src/test_rig_bluesky/plans.py

```diff
@@ -42,3 +43,34 @@ def spectroscopy(
     spec = spec or Line(sample_stage.x, 0, 5, 5)
 
     yield from spec_scan({spectroscopy_detector, sample_stage}, spec, metadata=metadata)
+
+
+def demo_spectroscopy(
+    spectroscopy_detector: AravisDetector = spectroscopy_detector,
+    sample_stage: XYZStage = sample_stage,
+    numbers_of_points: int = 5,
+    grid_size: float = 5.0,
+    grid_origin_x: float = 0.0,
+    grid_origin_y: float = 0.0,
+    exposure_time: float = 0.1,
+    metadata: dict[str, Any] | None = None,
+) -> MsgGenerator[None]:
+    """Spectroscopy plan intended for use in Visr demonstrations to visitors.
+    The time taken to scan is approximately linear in number_of_points.
```

Comment:
Nit: The argument is called `numbers_of_points`, not sure which one it is supposed to be. I'm also not sure either name is explicit enough, maybe `points_per_axis`? Or you could instead have a `resolution` parameter and perform the calculation with `grid_size`